packages/vite/src/node/plugins/worker.ts

```diff
@@ -92,24 +47,40 @@ export async function bundleWorkerEntry(
     },
     preserveEntrySignatures: false
   })
-  let chunk: Rollup.OutputChunk
+  let chunk: OutputChunk
   try {
+    const workerOutputConfig = config.worker.rollupOptions.output
+    const workerConfig = workerOutputConfig
+      ? Array.isArray(workerOutputConfig)
+        ? workerOutputConfig[0] || {}
+        : workerOutputConfig
+      : {}
```

Comment:
Oh I miss one point.
```js
Array.isArray(workerOutputConfig)
        ? workerOutputConfig[0] || {}
        : workerOutputConfig
```
It is always using the first item. Can we limit the type just accept array? @patak-dev

Reply:
I saw this and it makes sense in this context. We could warn if there are two output options maybe